sktime/classification/base.py

```diff
@@ -77,7 +77,7 @@ def fit(self, X, y):
         sets is_fitted flag to true
         """
 
-        coerce_to_numpy = self._all_tags()["coerce-X-to-numpy"]
+        coerce_to_numpy = self.get_class_tag("coerce-X-to-numpy", False)
```

Comment:
as discussed, here and throughout, I'd remove the setting of default values here and instead do that in the BaseClassifier (or corresponding base class)
```suggestion
        coerce_to_numpy = self.get_class_tag("coerce-X-to-numpy")
```

Reply:
ultimately, yes, very much agreed. But, I think the tag defaults are consistently set only in forecasting (where I've removed the defaults from `get_` etc). I wouldn't dare remove the defaults from classification until we've completed the interface refactor there...

Reply:
ok